make_ghpages/requirements.txt

```
@@ -1,2 +1,2 @@
 jinja2==3.0.0a1
-optimade~=0.10.0
+optimade>=0.12.0
```

There shouldn't be a reason to change this from `~` to `>`, but it doesn't matter to me, since this will, in theory, be more general, including also a future major version change.
```suggestion
optimade~=0.12.0
```

---

I switched to `>` so this repo can become more of a "set-and-forget"; it should always use the latest validator but 0.12 is the new minimum version. I don't think we want to have e.g. dependabot on this repo...